documentation/modules/overview/con-overview-components-cluster-operator.adoc

```diff
@@ -27,6 +27,11 @@ The Cluster Operator can also deploy (through configuration of the `Kafka` resou
 
 The Topic Operator and User Operator function within the Entity Operator on deployment.
 
+You can also configure the Cluster Operator to use the Strimzi Drain Cleaner.
+A deployment of the Strimzi Drain Cleaner helps with pod evictions.
+It annotates pods being evicted with a rolling update annotation.
+The annotation informs the Cluster Operator to perform the rolling update.
```

Comment:
I wanted to get a line in the overview on Drain Cleaner. I thought I'd add to this PR before it was cherry-picked to 0.26. Sorry for the confusion. That first line isn't right. Changed to:

> You can use the Cluster Operator with a deployment of Strimzi Drain Cleaner to help with pod evictions.
> By deploying the Strimzi Drain Cleaner, you can use the Cluster Operator to move Kafka pods instead of Kubernetes.
> Strimzi Drain Cleaner annotates pods being evicted with a rolling update annotation.
> The annotation informs the Cluster Operator to perform the rolling update.

Reply:
Yeah, ok ... lets have it here. But we need to fix it to make it correct.